tests/services/test_service_queryparser.py

```diff
@@ -25,22 +28,22 @@ def test_extra_params():
     """Test for the query parser."""
     p = QueryParser(system_identity).factory(fields=["title"])
     assert p(system_identity).parse("a").to_dict() == {
-        'query_string': {'query': "a", "fields": ["title"]}}
+        "query_string": {"query": "a", "fields": ["title"]}
+    }
 
 
 @pytest.mark.parametrize(
     "query",
     [
         "(new your city) OR (big apple)",
-        "\"a phrase\""
-        "status:active",
-        "author:\"John Smith\"",
+        '"a phrase"' "status:active",
+        'author:"John Smith"',
```

Comment:
I think there was a comma missing from the beginning as in:
```suggestion
        '"a phrase"' ,
        "status:active",
        'author:"John Smith"',
```